History.txt

```diff
@@ -1,3 +1,15 @@
+Changes between dev/ipv6_integration branch 4.0.0-RC3 and 4.0.0-RC2 releases:
+		+ Added memory safety proofs for IPv6 using[ CBMC automated reasoning tool ] ( https:/*www.cprover.org/cbmc/)
+		+ Added build separation in FreeRTOS+TCP stack to make the library more lightweight and optimized for users,
```

Comment:
Can we add something like this "Added build separation in FreeRTOS+TCP stack to make the library more modular. This enables the users to compile and create a low footprint binary with only the required functionalities."